File: src/main/java/com/nunchuk/android/type/SatsCardSlotStatusHelper.java
package com.nunchuk.android.type;

// This class should be kept in java since jni can't access object class
public final class SatsCardSlotStatusHelper {

    private SatsCardSlotStatusHelper() {
    }

    public static SatsCardSlotStatus from(final int ordinal) {
        return CollectionUtil.fromOrdinal(SatsCardSlotStatus.values(),
                ordinal,
                SatsCardSlotStatus.UNUSED
        );
    }

}

File: src/main/java/com/nunchuk/android/type/SatsCardSlotStatusHelper.kt
package com.nunchuk.android.type

object SatsCardSlotStatusHelper {
    @JvmStatic
    fun from(ordinal: Int): SatsCardSlotStatus =
        CollectionUtil.fromOrdinal(SatsCardSlotStatus.entries, ordinal, SatsCardSlotStatus.UNUSED)
}
